lib/tools/system-calls.js

```diff
@@ -608,4 +614,21 @@ systemCallMethods.ls = async function (remotePath) {
   }
 };
 
+systemCallMethods.fileSize = async function (remotePath) {
+  try {
+    let files = await this.ls(remotePath, ['-la']);
```

Comment:
We could also use the following cmdline to extract file size: `adb shell ls -la /sdcard/view.xml | tr -s ' ' | cut -d' ' -f 5`
All these shell commands are available even in Android 4.2